driver/src/main/java/org/neo4j/driver/v1/exceptions/AuthenticationException.java

```diff
@@ -25,7 +25,7 @@
  *
  * @since 1.1
  */
-public class AuthenticationException extends SecurityException
+public class AuthenticationException extends ClientException
```

Comment:
I am concerned about changing this from SecurityException to ClientException directly, as it might break any users who already moved to 1.2.0/1.2.1 users.

Can we make it both ClientException and SecurityException for Java and any other language that is sensitive to Exception types?